src/components/categories/restaurant/Restaurants.vue

```diff
@@ -63,13 +59,28 @@
                     style: style
                 })
             },
+            sendDataToChange(author, text,style) {
+                this.$emit('changeMessage', {
+                    author: author,
+                    data: text,
+                    style: style
+                })
+            },
             addCategoryDropdownOnClick() {
                 this.showCategoryDropdown1.push(true);
             },
             closeCategoryDropdownOnClick() {
                 this.showCategoryDropdown1.pop();
                 this.showCategoryDropdown1.push(false);
             },
+            cityDropdownOnClick(value){
+                this.showCityDropdown = false;
+                this.city = value;
+                this.sendMessage("user", `${this.$t('weather.user.chooseCity')} ${this.city.city}`);
```

Comment:
Zróbmy albo oddzielny message na to, albo zmieńmy klucz weather na jakiś ogólny np. user.chooseCity.